### PR TITLE
object_kinsta1_obj

### PR DESCRIPTION
**Pull request by hack-nuss**

### CODE REVIEW

**Comment by tom-overton** [comment]
For these ones that are just doors, they really should be named after their location in-game. This particular one is the door in the Swamp Spider House:
image

File: assets/xml/objects/object_kinsta1_obj.xml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ﻿<Root>
+    <!-- Object for Swamp Spider House Door -->
     <File Name="object_kinsta1_obj" Segment="6">
-        <DList Name="object_kinsta1_obj_DL_000190" Offset="0x190" /> <!-- Original name is "m2_KIN1_shutter_modelT" -->
-        <DList Name="object_kinsta1_obj_DL_000198" Offset="0x198" /> <!-- Original name is "m2_KIN1_shutter_model" -->
-        <Texture Name="object_kinsta1_obj_Tex_0002F0" OutName="tex_0002F0" Format="rgba16" Width="32" Height="64" Offset="0x2F0" />
+        <DList Name="gSwampSpiderHouseDoorEmptyDL" Offset="0x190" /> <!-- Original name is "m2_KIN1_shutter_modelT" -->
+        <DList Name="gSwampSpiderHouseDoorDL" Offset="0x198" /> <!-- Original name is "m2_KIN1_shutter_model" -->
```
Comment (engineer124) [inline assets/xml/objects/object_kinsta1_obj.xml:5]:
```suggestion
        <DList Name="gSwampSpiderHouseSlidingDoorDL" Offset="0x198" /> <!-- Original name is "m2_KIN1_shutter_model" -->
```
I would put the full `SlidingDoor` in there, just like `object_crace_object` i.e. `gDekuShrineSlidingDoorDL`. You've done a few other sliding door objects as well, this comment applies to those as well

File: assets/xml/objects/object_kinsta1_obj.xml [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
+        <Texture Name="gSwampSpiderHouseTex" OutName="swamp_spider_house_door" Format="rgba16" Width="32" Height="64" Offset="0x2F0" />
```
Comment (engineer124) [inline assets/xml/objects/object_kinsta1_obj.xml:6]: You missed the `Door` in the name, but it should also have `SlidingDoor` in there

File: assets/xml/objects/object_kinsta1_obj.xml [hunk 3]
```diff
@@ -1,7 +1,8 @@
 ﻿<Root>
+    <!-- Object for Swamp Spider Sliding House Door -->
```
Comment (engineer124) [inline assets/xml/objects/object_kinsta1_obj.xml:2]:
```suggestion
    <!-- Object for Swamp Spider House Sliding Door -->
```

File: assets/xml/objects/object_kinsta1_obj.xml [hunk 4]
```diff
@@ -1,7 +1,8 @@
...
     <File Name="object_kinsta1_obj" Segment="6">
-        <DList Name="object_kinsta1_obj_DL_000190" Offset="0x190" /> <!-- Original name is "m2_KIN1_shutter_modelT" -->
-        <DList Name="object_kinsta1_obj_DL_000198" Offset="0x198" /> <!-- Original name is "m2_KIN1_shutter_model" -->
-        <Texture Name="object_kinsta1_obj_Tex_0002F0" OutName="tex_0002F0" Format="rgba16" Width="32" Height="64" Offset="0x2F0" />
+        <DList Name="gSwampSpiderHouseSlidingDoorEmptyDL" Offset="0x190" /> <!-- Original name is "m2_KIN1_shutter_modelT" -->
+        <DList Name="gSwampSpiderHouseSlidingDoorDL" Offset="0x198" /> <!-- Original name is "m2_KIN1_shutter_model" -->
+        <Texture Name="gSwampSpiderHouseSlidingDoorTex" OutName="swamp_spider_house_door" Format="rgba16" Width="32" Height="64" Offset="0x2F0" />
```
Comment (engineer124) [inline assets/xml/objects/object_kinsta1_obj.xml:6]:
```suggestion
        <Texture Name="gSwampSpiderHouseSlidingDoorTex" OutName="swamp_spider_house_sliding_door" Format="rgba16" Width="32" Height="64" Offset="0x2F0" />
```

**Review by engineer124 — APPROVED** [review]

**Review by tom-overton — APPROVED** [review]

**Review by AngheloAlf — APPROVED** [review]

**Pull request merged**